common/src/test/java/com/alibaba/fescar/common/util/StringUtilsTest.java

```diff
@@ -41,10 +41,10 @@
      */
     @Test
     public void testIsEmpty() {
```

Comment:
`public void testIsEmpty()`
better to change the method name to testIsNullOrEmpty